### ISSUE TITLE
Sort the parameters in docs

### ISSUE THREAD

**Issue by fwkoch** [3ptscience/properties#58]
Should be able to alphabetize them programatically.

**Comment by fwkoch** [comment]
My thoughts on this: In the HasProperties metaclass have something like:

```
_prop_doc_order = classdict.pop('_prop_doc_order', sorted(_props))
```
Then sort `req`, `opt`, and `imm` based on this when adding to `doc_str`.

Then the props will be alphabetized unless `_prop_doc_order` is set in `HasProperties` subclass. There also needs to be some validation of this (ie make sure all props are included in `_prop_doc_order`).

**Comment by fwkoch** [comment]
Closed by #73.
